Index: app/src/main/java/net/laenredadera/app/android/lyricsradio/data/repositories/MediaInfoRepository.kt
===================================================================
diff --git a/app/src/main/java/net/laenredadera/app/android/lyricsradio/data/repositories/MediaInfoRepository.kt b/app/src/main/java/net/laenredadera/app/android/lyricsradio/data/repositories/MediaInfoRepository.kt
--- a/app/src/main/java/net/laenredadera/app/android/lyricsradio/data/repositories/MediaInfoRepository.kt	
+++ b/app/src/main/java/net/laenredadera/app/android/lyricsradio/data/repositories/MediaInfoRepository.kt	
@@ -19,7 +19,7 @@
     }
 
     suspend fun getArtistMbId(artistName: String, trackName: String): String {
-        return lastFM.getArtistMbId(artistName, trackName)
+        return lastFM.getAlbumMbId (artistName, trackName)
     }
 
     suspend fun getAlbumInfoWiki(mbid: String): Wiki {
